core/modules/replica/HttpProcessor.cc

```diff
@@ -278,6 +279,22 @@ void HttpProcessor::_initialize() {
                         "SELECT-QUERY-BY-ID");
             }
         },
+        {"GET", "/replication/sql/table/schema",
+            [self](qhttp::Request::Ptr const& req, qhttp::Response::Ptr const& resp) {
+                HttpSqlSchemaModule::process(
+                        self->controller(), self->name(), self->_processorConfig,
+                        req, resp,
+                        "GET-TABLE-SCHEMA");
+            }
+        },
+        {"PUT", "/replication/sql/table/schema",
+            [self](qhttp::Request::Ptr const& req, qhttp::Response::Ptr const& resp) {
+                HttpSqlSchemaModule::process(
+                        self->controller(), self->name(), self->_processorConfig,
+                        req, resp,
+                        "ALTER-TABLE-SCHEMA", HttpModule::AUTH_REQUIRED);
+            }
+        },
```

Comment:
Maybe this call to httpServer()->addHandlers should be broken into smaller pieces?